src/containers/UserEdit.jsx

```diff
@@ -252,7 +257,8 @@ UserEdit.propTypes = {
   nextUrl: PropTypes.string,
   style: PropTypes.string,
   handleClose: PropTypes.func,
-  openSuccessDialog: PropTypes.func
+  openSuccessDialog: PropTypes.func,
+  query: PropTypes.function
```

Comment:
what/where is this query property?  the query result will end up in the `data` property above (I believe).  I think this is unnecessary.

Reply:
I think you are right. Don't think we are using it inside the component. Will review it.